### ISSUE TITLE
Swift 6 Language Mode

### ISSUE THREAD

**Issue by dehlen** [sindresorhus/KeyboardShortcuts#180]
First of all thanks for the library and the ongoing maintenance. 

I am currently looking at the Swift 6 Language mode. I recognise there a ways to import this library without any issue even when using Swift 6 Language mode. However when building the package itself I found a few errors are thrown. Any plan to lift this packages language mode to 6 and to fix those errors?

Kind regards,
David

**Comment by dehlen** [comment]
Yes as stated above the package works just fine even in a Swift 6 app which depends on this package. My point was rather when adding `swiftLanguageVersions: [.version("6")]` to the package manifest and building the package directly it encounters various compiler errors. Of course since one can use the package just fine and Xcode 16 isn't even officially released I recognise the fact that this has no priority. But I think it would be nice to have a look at those compiler warnings and fix/or silence them when appropriate once Xcode 16 is released. Feel free to either close due to the timeframe or keep it open as a reminder to have a look at this at a later point.

**Comment by godbout** [comment]
definitely getting issues in some cases:

<img width="1273" alt="Image" src="https://github.com/user-attachments/assets/df18db35-389f-4d03-bff6-72f4fdf359f1" />
